rq/results.py

```diff
@@ -181,7 +181,11 @@ def serialize(self):
         if self.exc_string is not None:
             data['exc_string'] = b64encode(zlib.compress(self.exc_string.encode())).decode()
 
-        serialized = self.serializer.dumps(self.return_value)
+        try:
+            serialized = self.serializer.dumps(self.return_value)
+        except:  # noqa
+            return data
```

Comment:
ℹ️ I'm returning early so we don't b64encode a null value, which has no real sense here